pkg/cache/cache.go

```diff
@@ -30,41 +32,68 @@ func New(cfg *config.Config) BundleCache {
 // FindBundle looks for a given bundle tag in the Porter bundle cache and
 // returns the path to the bundle if it exists. If it is not found, an
 // empty string and the boolean false value are returned.
-func (c *cache) FindBundle(bundleTag string) (string, bool, error) {
+func (c *cache) FindBundle(bundleTag string) (string, string, bool, error) {
```

Comment:
nit: perhaps update comment to mention that this function now also may return the relocation-mapping file path, if exists